TeamSnapSDK/SDK/DataTypes/TSDKBroadcastEmail.m

```diff
@@ -42,6 +42,7 @@ - (instancetype)initWithDivisionID:(NSString *)divisionId teams:(NSArray <TSDKTe
         [super setString:body forKey:@"body"];
         [super setString:subject forKey:@"subject"];
         [super setString:[sender memberId] forKey:@"member_id"];
+        [super setBool:NO forKey:@"is_draft"];
```

Comment:
Can we check if it is set before forcing it in case the client wants to change it?